spec/controllers/coronavirus_form/basic_care_needs_controller_spec.rb

```diff
@@ -3,11 +3,14 @@
 require "spec_helper"
 
 RSpec.describe CoronavirusForm::BasicCareNeedsController, type: :controller do
+  include_examples "redirections"
+
   let(:current_template) { "coronavirus_form/basic_care_needs" }
   let(:session_key) { :basic_care_needs }
 
   describe "GET show" do
```

Comment:
Should all these get an additional test, something like:

```ruby
it "redirects to the first question if the first question is unanswered" do
  session[:live_in_england] = nil
  get :show
  expect(response).to redirect_to(live_in_england_path)
end
```

Reply:
Yup - so've moved them out in to a shared example and they are included via `include_examples "redirections"`

Reply:
Ah ha!